<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-07-24T14:06:38 IST" name="Default suite" finished-at="2021-07-24T14:06:49 IST" duration-ms="11316">
    <groups>
    </groups>
    <test started-at="2021-07-24T14:06:38 IST" name="Default test" finished-at="2021-07-24T14:06:49 IST" duration-ms="11316">
      <class name="testCases.LoginTest">
        <test-method is-config="true" signature="testSetUp()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:38 IST" name="testSetUp" finished-at="2021-07-24T14:06:48 IST" duration-ms="9968" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetUp -->
        <test-method is-config="true" signature="methodSetUp()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:48 IST" name="methodSetUp" finished-at="2021-07-24T14:06:48 IST" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodSetUp -->
        <test-method signature="clickMenu()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:48 IST" name="clickMenu" finished-at="2021-07-24T14:06:49 IST" duration-ms="1182" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickMenu -->
        <test-method is-config="true" signature="methodTearDown()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:49 IST" name="methodTearDown" finished-at="2021-07-24T14:06:49 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodTearDown -->
        <test-method is-config="true" signature="methodSetUp()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:49 IST" name="methodSetUp" finished-at="2021-07-24T14:06:49 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodSetUp -->
        <test-method signature="loginTitleTest()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:49 IST" name="loginTitleTest" finished-at="2021-07-24T14:06:49 IST" duration-ms="11" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at testCases.LoginTest.loginTitleTest(LoginTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTitleTest -->
        <test-method is-config="true" signature="methodTearDown()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:49 IST" name="methodTearDown" finished-at="2021-07-24T14:06:49 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodTearDown -->
        <test-method is-config="true" signature="testTearDown()[pri:0, instance:testCases.LoginTest@485966cc]" started-at="2021-07-24T14:06:49 IST" name="testTearDown" finished-at="2021-07-24T14:06:49 IST" duration-ms="78" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTearDown -->
      </class> <!-- testCases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
